02-react-axe.md

```diff
@@ -0,0 +1,37 @@
+ # 02. Use react-axe to Audit Accessibility Issues at Runtime during Development
+
+**[📹 Video](https://egghead.io/lessons/react-use-react-axe-to-audit-accessibility-issues-at-runtime-during-development)**
+
+**[💻 CodeSandbox](https://codesandbox.io/s/gallant-hugle-4xqre?from-embed)**
+
+
+Install `react-axe` plugin:
+
+`npm install --save-dev react-axe`
+
+* To initialize the plugin, you'll have to add it to the file that starts your application (typically something like `app.js` or `index.js`).
+
+* Initialize it before rendering your first React component and make sure it runs only in development (to avoid performance overhead and console.logs in the production). 👍
```

Comment:
```suggestion
* 👍Initialize it before rendering your first React component and make sure it runs only in development (to avoid performance overhead and console.logs in the production). 
```